x-pack/plugins/index_management/public/application/sections/enrich_policy_create/steps/fields/indices_selector.tsx

```diff
@@ -77,8 +77,10 @@ export const IndicesSelector = ({ field, ...rest }: Props) => {
 
   // Load first 10 indices on mount so that the ComboBox has some options
   useEffect(() => {
-    onSearchChange('*');
-  }, [onSearchChange]);
+    if (isEmpty(field.value)) {
+      onSearchChange('*');
```

Comment:
We could also add this small change to `onSearchChange()` to ensure that the dropdown arrow is also displayed when there are selected options and the user hasn't typed in anything:

```
const onSearchChange = useCallback(
    async (search: string) => {
      const indexPattern = isEmpty(search) ? '*' : search;
      setIsIndiciesLoading(true);
      setIndexOptions(await getIndexOptions(indexPattern));
      setIsIndiciesLoading(false);
    },
    [setIsIndiciesLoading, setIndexOptions]
 );
```

<img width="436" alt="Screenshot 2023-09-07 at 11 10 36" src="https://github.com/elastic/kibana/assets/59341489/2947d9da-9d80-4acb-b7e3-015c2cc69b76">